src/modal_analysis.cu

```diff
@@ -416,8 +468,9 @@ void MODAL_ANALYSIS::preprocess(char *input_dir, Atom *atom)
         // Setup binning
         if (f_flag)
         {
-            double *f;
-            CHECK(cudaMallocManaged((void **)&f, sizeof(double)*num_modes));
+            GPU_Vector<double> f;
```

Comment:
These three lines can be written as 
GPU_Vector<double> f(num_modes, Memory_Type::managed);

Reply:
Sorry, GPU_Vector<double> f(num_modes, Memory_Type::managed);

Reply:
Good idea. I'm going to simplify some of the resize calls too.